project/ws/app/src/lib/routes/user-profile/components/user-profile/user-profile.component.ts

```diff
@@ -1753,14 +1753,14 @@ export class UserProfileComponent implements OnInit, OnDestroy {
           formdata.append('data', result)
           
           // Working on Upload Profile image, still testing the API.
-          // this.userProfileSvc.updateProfilePic(formdata).subscribe(
-          //   (res: any) => {
-          //     console.log('res - ', res);
-          //   },
-          //   (error: HttpErrorResponse) => {
-          //     console.log("error - ", error);  
-          //   }
-          // );
+          this.userProfileSvc.updateProfilePic(formdata).subscribe(
+            (res: any) => {
+              console.log('res - ', res);
+            },
+            (error: HttpErrorResponse) => {
```

Comment:
Please check console should not break the build

Reply:
API is working now, @bharathravi-in  is working on further URL (in discuss with Back-end team)